Feat: Support bluetooth device management

sudo-Tiz opened the pull request:
## Enhance Bluetooth Quick Settings with Full Device Management

Brings Bluetooth functionality to feature parity with WiFi, enabling complete device management from the quick settings panel.

### Features

**Device Display**
- Shows connected device names on the Bluetooth button (single name or "N devices")
- Battery indicators for connected devices

**Discovery & Pairing**
- Scan for nearby devices with toggle button (Refresh/Stop)
- Auto-stop after 15 seconds to save battery
- One-click pairing for new devices

**Connection Management**
- Three sections: Connected, Paired, and Available devices
- Connect/disconnect with single click
- Scrollable device lists with battery status

### Technical Changes
- Extended `BluetoothDevice` structure with connection state fields
- Added DBus methods for discovery, pairing, and connection management
- Redesigned UI with organized device sections

MalpenZibo commented:
Wooo ohh this is huge! I was planning this feature for I while but then I decided to implement the calendar and meteo :)

I will review this PR as soon as I can

MalpenZibo commented on src/modules/settings/bluetooth.rs:
```diff
@@ -128,41 +189,167 @@ impl BluetoothSettings {
+                                row![
+                                    text(d.name.clone()).width(Length::Fill),
+                                    text("Pair").size(12),
+                                ]
```
```suggestion
                                ]
                                .align_y(Vertical::Center)
```

sudo-Tiz commented:
Thank you for your sharp eyes. I fixed the minor issue you pointed out and added the remove button. For the auth prompt part, I’d prefer to add it later in a separate PR.

MalpenZibo reviewed (approved):
🚀
